rust-runtime/aws-smithy-types/Cargo.toml

```diff
@@ -11,17 +11,30 @@ license = "Apache-2.0"
 repository = "https://github.com/awslabs/smithy-rs"
 
 [features]
+rt-tokio = ["dep:tokio-util", "dep:tokio", "tokio?/rt", "tokio?/fs", "tokio?/io-util", "tokio-util?/io"]
 test-util = []
 serde-serialize = []
 serde-deserialize = []
 
 [dependencies]
 base64-simd = "0.8"
+bytes = "1"
+bytes-utils = "0.1"
+http = "0.2.3"
+http-body = "0.4.4"
+hyper = "0.14.26"
```

Comment:
Why is hyper in here?

Reply:
For `impl From<hyper::Body> for SdkBody (src/byte_stream.rs:471)`

Reply:
@rcoh, do we actually need this?

Reply:
Looks like it's at least used by the server (if we remove it, the server codegen fails as follows):
```
     --> rest_json/rust-server-codegen/src/operation.rs:35849:5
      |
35849 | /     ::aws_smithy_http_server::request::FromRequest<
35850 | |         ::aws_smithy_http_server::protocol::rest_json_1::RestJson1,
35851 | |         B,
35852 | |     > for crate::input::StreamingTraitsInput
      | |_____^     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
35857 |       B: Into<::aws_smithy_http::byte_stream::ByteStream>,
      |          ------------------------------------------------ unsatisfied trait bound introduced here
```

Reply:
If it's only for a conversion in server, we can just change it to not use a From trait for that conversion.

Reply:
`impl From<hyper::Body> for SdkBody` will stay in `aws-smithy-types` but with a feature gate. By default, there is no conversion available for `hyper::Body` -> `SdkBody`; for those who want that conversion, they need to opt-in to enable it by that cargo feature. Once hyper goes 1.0, we can add a new conversion backwards-compatible manner.

We keep track via https://github.com/awslabs/smithy-rs/issues/3033, and will work on it right after this PR.